HW_module_6.py

```diff
@@ -0,0 +1,155 @@
+import datetime
```

Comment:
!IMPORTANT!
Define your input format (one or many records).
You need to implement a solution which provides the possibility to load 1 newsfeed or multiple newsfeeds recorded into your final newsfeed file.